app/Http/Requests/Admin/AdminFormRequest.php

```diff
@@ -44,9 +44,6 @@ public function authorize()
      */
     public function normalize($only = [])
     {
-        return array_merge(
-            $this->only($only),
-            $this->intersect(array_keys($this->rules()))
-        );
+        return $this->all(empty($only) ? array_keys($this->rules()) : $only);
```

Comment:
This could be written better.

Reply:
Yeah, but thats what happen when they deprecate `$request->intersect()` and make all work like it if you pass in keys.

I'll probably end up rewriting that at some point, but need better test coverage first because that controls a lot of functionality.